docs/tutorial/deployment.md

```diff
@@ -212,18 +212,28 @@ ln -s /usr/bin/wslview /usr/local/bin/xdg-open
 If you successfully install flyctl but get an error saying "`fly` is not recognized" or "flyctl: command not found error", then you must add flyctl to your PATH. For more information, read [Getting flyctl: command not found error post install](https://community.fly.io/t/getting-flyctl-command-not-found-error-post-install/4954/1).
 ```
 
-8. Create your Fly.io project by running `fly launch` and answering the resulting prompt questions as follows:
+8. Create your Fly.io project by running `fly launch`. Then press `y` to configure the settings.
+9. You will be taken to an admin screen on fly.io. Fill out the fields as follows:
+
+| Field                                                     | Instruction                                                                          |
+|-----------------------------------------------------------|--------------------------------------------------------------------------------------|
+| Choose a region for deployment                            | Select the region closest to the _AWS_S3_REGION_NAME_ in your _env.production_ file. |
+| CPU & Memory                                              | VM Size - shared-cpu-1x  VM Memory - 512 MB                                          |
+| Database                                                  | Fly Postgres - choose smallest option                                                |
+
+click confirm **Confirm settings**
+
+```{note}
+Not creating the database directly with the application leads to the app and the database not directly connected. 
+So if the app is going to launched again, the easiest solution is to create a new database directly
```

Comment:
Question: I'm not quite sure what you're trying to say here?

Reply:
During the launch of the app, the web UI offers an option to create a associated postgresdb alongside the app. If for some reason the launch and deployment of the app didn't go as planned and you want to redeploy the app its best to create a new postgres db alongside the new app so that the app and db are connected. 
If a user relaunches/redeploys the app without a db the app and the database has to be connected using additional commands and configuration which is perhaps a little to much hassle for a deployment. 
So if want to launch a new app, create a new db alongside for better out of the box config. 

Now time for ChatGPT to shorten that to some small note 😄

Reply:
Perhaps reprhase it as:

> Not creating the database directly with the application leads to the app and the database not connected. 
> So if the app is going to launched again using `fly launch`, the easiest solution is to create a new database with the launch of the app through the web UI.

Reply:
That wording feels a bit nicer :slightly_smiling_face: